config/locales/en.yml

```
@@ -1327,6 +1327,8 @@ en:
       title: "Add registration for %{comp}"
       info: "This will create an approved registration. Use this form to add walk-ins during the competition."
       add: "Add registration"
+      ots_not_enabled: "Registration is closed and on-the-spot registration is not enabled - a Delegate must enable it from the Organizer View"
```

The message should not be restrictive about where or how you make the edits. For what it's worth, you _could_ email WCAT, get lucky that they read your mail within 10 minutes, and get it fixed that way. And that would (theoretically) work.

```suggestion
      ots_not_enabled: "Registration is closed and on-the-spot registration is not enabled - you must edit the competition to enable it first"
```